### PR TITLE
add addNugetPackageReference to vscode-mssql.d.ts

### PR DESCRIPTION
**Pull request by kisantia**
This was added in ADS in mssql.d.ts in https://github.com/microsoft/azuredatastudio/pull/22757. After this is merged, I'll update the corresponding vscode-mssql.d.ts file in ADS.

### CODE REVIEW

**Review by SakshiS-harma — APPROVED** [review]

**Review by Benjin — APPROVED** [review]

**Comment by Benjin** [comment]
Be sure to update `/src/services/sqlProjectsService.ts` as well for the PR builds to pass!

File: typings/vscode-mssql.d.ts [hunk 1]
```diff
@@ -485,6 +485,19 @@ declare module 'vscode-mssql' {
...
+		 * @param projectUri Absolute path of the project, including .sqlproj
+		 * @param packageName Name of the referenced nuget package
+		 * @param packageVersion Version of the referenced nuget package
+		 * @param suppressMissingDependencies Whether to suppress missing dependencies
+		 * @param databaseVariable SQLCMD variable name for specifying the other database this reference is to, if different from that of the current project
+		 * @param serverVariable SQLCMD variable name for specifying the other server this reference is to, if different from that of the current project.
+			 If this is set, DatabaseVariable must also be set.
+		 * @param databaseLiteral Literal name used to reference another database in the same server, if not using SQLCMD variables
+		 */
+		addNugetPackageReference(projectUri: string, packageName: string, packageVersion: string, suppressMissingDependencies: boolean, databaseVariable?: string, serverVariable?: string, databaseLiteral?: string): Promise<ResultStatus>;
```
Comment (Benjin) [inline typings/vscode-mssql.d.ts:499]: Be sure to update `/src/services/sqlProjectsService.ts` as well for the PR builds to pass!

Comment (kisantia) [reply typings/vscode-mssql.d.ts:499]: whoops that's what happens when I don't build locally before sending the PR

**Review by Benjin — APPROVED** [review]

File: typings/vscode-mssql.d.ts (hunk 1 above)
Comment (Benjin) [reply typings/vscode-mssql.d.ts:499]: Man, who would ever do that. 😇

**Pull request merged**